src/backend/core/api/viewsets.py

```diff
@@ -399,3 +400,80 @@ def perform_update(self, serializer):
                 raise exceptions.ValidationError({"role": message})
 
         serializer.save()
+
+
+class InvitationViewset(
```

Comment:
Should I be allowed to update an invitation in the admin ?

<img width="1032" alt="Capture d’écran 2024-03-05 à 12 36 13" src="https://github.com/numerique-gouv/people/assets/45729124/94d32b8d-3a85-4bf1-b803-c07d0e13e6df">

Reply:
I don't think so. I'll make all these fields `read_only` in the admin.

Reply:
✅ ! I marked all fields as read_only if the object already exists, and removed save buttons. I made sure we can still delete objects

I stole everything from this thread https://stackoverflow.com/questions/4343535/django-admin-make-a-field-read-only-when-modifying-obj-but-required-when-adding